R/choices_selected.R

```diff
@@ -34,8 +34,8 @@ no_select_keyword <- "-- no selection --"
 #' be on top of the drop-down field.
 #' @param fixed (optional `logical`) Whether to block user to select choices.
 #'
-#' @return `choices_selected` returns object of class `choices_selected` and of
-#' type list which contains the specified `choices`, `selected`, `keep_order` and `fixed`.
+#' @return `choices_selected` list object, encapsulating the specified
```

Comment:
It's repetitive, but given it appears alongside `is.choices_selected, it seems necessary. (if you go with 3. of my other comment, then apply it here too)

```suggestion
#' @return `choices_selected` returns list of `choices_selected`, encapsulating the specified
```

![image](https://github.com/insightsengineering/teal.transform/assets/211358/bc013d05-2da2-409b-a042-4e9f0706d238)